lib/Dialect/FIRRTL/Ops.cpp

```diff
@@ -58,6 +58,25 @@ static ParseResult parseCircuitOp(OpAsmParser &parser, OperationState &result) {
   return success();
 }
 
+static LogicalResult verifyCircuitOp(CircuitOp &circuit) {
+  StringRef main = circuit.name();
+
+  // Check that the circuit has a non-empty name
```

Comment:
Super super nit-pick, but please end sentences in comments with a period.

Reply:
Fixed this comment and one other comment I added in the subsequent commit.